src/systems/ec/implementations/react/templates/pages/commemorative-coin/src/CommemorativeCoinPage.jsx

```diff
@@ -0,0 +1,408 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+import Button from '@ecl/ec-react-component-button';
+import Link from '@ecl/ec-react-component-link';
+import Pagination from '@ecl/ec-react-component-pagination';
+import Select from '@ecl/ec-react-component-select';
+import TextInput from '@ecl/ec-react-component-text-input';
+
+// Templates
+import SiteHeaderCore from '@ecl/ec-react-component-site-header-core';
+import SiteHeaderHarmonised from '@ecl/ec-react-component-site-header-harmonised';
+import SiteHeaderStandardised from '@ecl/ec-react-component-site-header-standardised';
+import PageHeaderCore from '@ecl/ec-react-component-page-header-core';
+import PageHeaderHarmonised from '@ecl/ec-react-component-page-header-harmonised';
+import PageHeaderStandardised from '@ecl/ec-react-component-page-header-standardised';
+import FooterCore from '@ecl/ec-react-component-footer-core';
+import FooterHarmonised from '@ecl/ec-react-component-footer-harmonised';
+import FooterStandardised from '@ecl/ec-react-component-footer-standardised';
+
+const CommemorativeCoinPage = ({
+  siteHeader,
+  pageHeader,
+  footer,
+  template,
+}) => (
+  <Fragment>
+    {template === 'core' && (
+      <Fragment>
+        <SiteHeaderCore {...siteHeader} data-ecl-auto-init="SiteHeaderCore" />
+        <PageHeaderCore {...pageHeader} />
+      </Fragment>
+    )}
+    {template === 'standardised' && (
+      <Fragment>
+        <SiteHeaderStandardised
+          {...siteHeader}
+          data-ecl-auto-init="SiteHeaderStandardised"
+        />
+        <PageHeaderStandardised {...pageHeader} />
+      </Fragment>
+    )}
+    {template === 'harmonised-g1' && (
+      <Fragment>
+        <SiteHeaderHarmonised
+          {...siteHeader}
+          data-ecl-auto-init="SiteHeaderHarmonised"
+          className="ecl-site-header-harmonised--group1"
+        />
+        <PageHeaderHarmonised {...pageHeader} />
+      </Fragment>
+    )}
+    {template === 'harmonised-g2' && (
+      <Fragment>
+        <SiteHeaderHarmonised
+          {...siteHeader}
+          data-ecl-auto-init="SiteHeaderHarmonised"
+          className="ecl-site-header-harmonised--group2"
+        />
+        <PageHeaderHarmonised {...pageHeader} />
+      </Fragment>
+    )}
+    <div className="ecl-u-pv-3xl">
+      <div className="ecl-container">
+        <div className="ecl-row">
+          <aside className="ecl-col-12 ecl-col-lg-3">
+            <h2 className="ecl-u-type-heading-2 ecl-u-type-color-black ecl-u-d-lg-none ecl-u-mv-none">
+              Coins (324)
+            </h2>
+            <h3 className="ecl-u-type-heading-3 ecl-u-type-color-black ecl-u-mt-l ecl-u-mt-lg-none">
```

Comment:
All the filters block is not clear for me, according to mocks, this block should be hidden for mobile and have the possibility of expanding it, via buttons "_Filter by_" / "_Hide the filters_" is this something we need to implement? There is also no submit button on the mocks..which I also find strange.

Reply:
Filters are following the pattern of the Search: https://ec.europa.eu/component-library/ec/content-types/search/usage/

Behaviors and visibility is "inheriting" this.